code-goodbuy/goodbuy-nextjs
# Barcode scanner

To test features
- for dependencies: `npm i` 
- to run local: `npm run dev`
- to run test and observe: `npm run test -- --watch`

1. Everything around scanner component is bundled in one file with test - still not sure it's the best approach, if it needs splitting, I'll work on it
2. You can't find scanner element in the page at the moment, It will be decided later
3. Test might complain that it can't read the camera list due to browser behavior.. that's okay.. it's fine on https but can't figure out how to deal with it gracefully for now :(
